### ISSUE TITLE
Does this api work at all?

### ISSUE THREAD

**Issue by zvodd** [StreamElements/api-docs#1]
Me attempting to hit the ends points for a response.
```
import requests
channel = "<CHANNEL_ID"
token = "<TOKEN>"
headers = {
    "Authorization": "Bearer {token}".format(token=token)
}

def hit_endpoint(channel, end_point, headers):
    resp = requests.get(
        "https://api.streamelements.com/kappa/v2/songrequest/{channel}/{ep}".format(channel=channel, ep=end_point), headers=headers)

    # filename = "{ep}_{sc}".format(ep=end_point, sc=resp.status_code)
    print()
    print("#######################################")
    print(end_point)
    print("#######################################")
    print(resp.request.headers)
    print("#######################################")
    print(resp.status_code)
    print("#######################################")
    print(resp.headers)
    print("#######################################")
    print(resp.content)


for e in ["player", 'pending', 'settings']:
    hit_endpoint(channel, e, headers)
```
all the end points either status code 500 or timeout for me

**Comment by heyanush** [comment]
Are endpoints other than songrequest working for you?

**Comment by zvodd** [comment]
Nope, I guess I might be doing something else wrong, but it seems to be the correct JWT token as I'm not getting 401 errors (changing token to invalid token does give 401 errors)

**Comment by heyanush** [comment]
Before going into the code, check the APIs working using Postman or some other tools. Let me know how that goes because everything working fine for me

**Comment by heyanush** [comment]
I generally get 500 error when there is a problem in request body. First try some simpler endpoints like 
`https://api.streamelements.com/kappa/v2/channels/me `
and see the response

**Comment by heyanush** [comment]
Basically there are two ways to find channel ID :
1. Using `https://api.streamelements.com/kappa/v2/channels/me ` API. It response you with list of channels with their ID.
2. Go to `https://streamelements.com/dashboard/account/channels` using user browser and there you can see Account ID.

I don't know why you were inspecting and all.

**Comment by zvodd** [comment]
The reason...
I'm not sure if you've read the documentation that in this repo and mirrored at `https://developers.streamelements.com/`. But most of the pages are blank or include non-relevant information such as the "getting started page" that suggests "to get super poweres" `$ give me super-powers`. As oppose to mentioning what common API variables are or what headers are required for Authenticated requests, or where to even find my Channel ID.

Given the state of the docs. I am genuinely curious why you would think anyone would be to doing anything but inspecting and attempting to reverse the API?

I deducted most information so far, from error messages from the REST API endpoints and it seems like it is STILL the way forward for figuring out the rest of REST API.

**Comment by heyanush** [comment]
Everything cool bud and yes, I wish I could say that to you :-) but anyways I am interested in your application, let me know your social handle I'll hit you up there.

**Comment by krtschmr** [comment]
the api docs are so bad, i don't want to live on this planet anymore
